### ISSUE TITLE
Editor.js:75 Uncaught Error: Expected target ref

### ISSUE THREAD

**Issue by gergana95** [tinymce/tinymce-react#183]
Hello, I keep getting this error on initializing: 
<img width="457" alt="Screen Shot 2020-10-20 at 9 37 38 AM" src="https://user-images.githubusercontent.com/29689645/96549276-f396e600-12b7-11eb-9eca-210fae24aad2.png">

This is the row which is getting the error from the library: 
`var target = _this.elementRef.current;`         

Here is my Editor component:
```
 <Editor
   apiKey="apiKey"
   inline
   init={{
      menubar: false,
      toolbar: false
   }}
   onEditorChange={setEditorState}
   value={editorState}
 /> 
```
initialValue is passed after data fetch in the same component with useEffect lifecycle hook. I am not sure if I am missing something that I should add as props. Did anybody else had the same issue?

**Comment by jdufresne** [comment]
I started experiencing the same error after upgrading from 3.7.0 to 3.8.0.

**Comment by gergana95** [comment]
@jdufresne I was using 3.8.0 and changing it to 3.7.0 worked for me. Thanks! I will use this version until problem is resolved.

**Comment by tiny-james** [comment]
Thanks for the error report. I'm investigating why the ref can be undefined in componentDidMount and I will hopefully have a patch out today.

**Comment by tiny-james** [comment]
I tried to create a replication using the details provided but I have not been able to replicate the error that way:
[https://codesandbox.io/s/tinymcereact-forked-wlzz0?file=/src/index.js](https://codesandbox.io/s/tinymcereact-forked-wlzz0?file=/src/index.js)

If you have more details that would help to replicate the error they would be appreciated.

I suspect the problem is that on the first editor created tinymce is not available and must be loaded. While that happens the first editor is unmounted and a new editor is created to replace it. That means when tinymce loads and tries to initialize the editor for the unmounted component it finds that the ref for the target element is null and it throws that error.

I will change the script to simply return when the ref is not set.

**Comment by tiny-james** [comment]
I have opened a PR that I hope will address this issue:
https://github.com/tinymce/tinymce-react/pull/184

Note that I would still like more details (preferably a replication case) to confirm that this does fix the issue.

**Comment by tiny-james** [comment]
Ok, the PR is merged. Let me know if this resolves the problem.

**Comment by gergana95** [comment]
Thanks for the quick fix, it is working now with version 3.8.1. I couldn't reproduce the issue in a codesandbox, but I investigated if furthermore and found out my component was rerendering 6 times from a HOC I created ( which is fetching data and supposed to not render if it is fetching).

I am thinking that one of the times when the component rerenders the library throws the error because the component hasn't been unmounted. I have to refactor my HOC but really it is now working without this issue even if component still rerenders.

@jdufresne You can check if you have a similar issue.

**Comment by jdufresne** [comment]
Confirmed, fixed in 3.8.1 :tada: Thanks for the quick fix!

**Comment by tiny-james** [comment]
Thanks for the confirmation. Closing this issue as "fixed".
